examples/kfto-kueue-sft-trainer.yaml

```diff
@@ -0,0 +1,83 @@
+# Example ConfigMap running fine tuning on bloom model
+# with twitter complaints dataset on 2 GPUs
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+data:
+  config.json: |
+    {
+      "model_name_or_path": "bigscience/bloom-560m",
+      "training_data_path": "/data/twitter_complaints.json",
+      "output_dir": "/data/output/tuning/bloom-twitter",
+      "tokenizer_name_or_path": "bigscience/bloom"
```

Comment:
Comma missing here after bloom, and extra comma after output.

Reply:
good catch! tokenizer should be removed anyways